### DISCUSSION TITLE
"The keys of Map must be non-nullable Strings" - is this still true?

### DISCUSSION THREAD

**Discussion by jason-downs** [serverpod/serverpod#2823] [Q&A] [answered]
In the "Get Started" section of the docs (but not in the "Models" section as far as I can tell), there is the following sentence:

"The keys of Map must be non-nullable Strings."

I know this used to be the case, but I thought that this had been changed several months ago in a newer version of serverpod and now is no longer true. Is this documentation correct, or should it be updated - or is it indeed the case that the map keys must still be Strings and I was mistaken?

**Comment by Isakdl** [comment, marked as answer]
You are right, Maps now supports any type that is generally supported by the Serverpod serialization. The documentation is outdated.
